test/puppetlabs/puppetdb/cli/services_test.clj

```diff
@@ -482,7 +463,9 @@
   (with-unconnected-test-db
     (let [config (-> (create-temp-config)
                      (assoc :database *db*)
-                     (assoc-in [:database :gc-interval] "0.01"))
+                     (assoc-in [:database :gc-interval] "0.01")
+                     (assoc-in [:database :report-ttl] "10d")
```

Comment:
Should be able to pull out the updates to the ttls here when we redef the func in the comment below.